### PR TITLE
Add finished lab4

### PR DESCRIPTION
**Pull request by malinastrand**

### CODE REVIEW

File: StringSearch.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                if(command.length == 3){
+                    try {
+                        String pattern = command[1];
+                        String fileName = command[2];
+                        File file = new File(fileName);
+                        Scanner fileScanner = new Scanner(file);
+
+                        while(fileScanner.hasNextLine()){
+                            String line = fileScanner.nextLine();
+                            if(line.contains(pattern)){
```
Comment (Coolakatter) [inline StringSearch.java:27]: A suggestion is that you could make pattern and line toLowerCase for a broader search.

File: StringSearch.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class StringSearch {
+
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        boolean quit = false;
```
Comment (Coolakatter) [inline StringSearch.java:9]: Perhaps another name than quit, that makes it more clear what the program do.

File: StringSearch.java (hunk 1 above)
Comment (malinastrand) [reply StringSearch.java:27]: Very good input, will fix

File: StringSearch.java (hunk 2 above)
Comment (malinastrand) [reply StringSearch.java:9]: Hm, I disagree,I think quit is good enough.

**Review by Coolakatter — APPROVED** [review]
Now it looks good!

**Pull request merged**